WordPress/gutenberg
# [RNMobile] Create drafts from template previews

`Guntenberg-mobile` PR -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/1730

## Description
This PR adds a new button in the `ModalHeaderBar` component to apply the selected layout from the preview modal.

## How has this been tested?
- Open WP iOS / WP Android
- Go to **Site Pages**
- Add a new page (for iOS Tap on the Add button in the header top right)
- Select a layout from the bottom picker (About / Contact)
- When the preview modal opens, click on the right button **Apply**
- **Expect** the layout to be added in the editor and the preview modal to be closed.

## Screenshots 
| iOS | Android |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/72145009-e1d3fa00-3399-11ea-97f6-85705c8f7d6d.gif" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/72145312-89e9c300-339a-11ea-8a0a-d431b347501b.gif" width=250 />  |

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
